app/routes/users/components/UserConfirmation.tsx

```diff
@@ -67,9 +67,9 @@ const UserConfirmationForm = () => {
 
   const dispatch = useAppDispatch();
 
-  const onSubmit = (data) =>
-    token &&
-    dispatch(createUser(token, data)).then((action) => {
+  const onSubmit = (data: FormValues) => {
+    if (!token) return;
+    return dispatch(createUser(token, data)).then((action) => {
```

Comment:
What difference does this make from the previous code?

Reply:
Doesn't really make any difference, but fixes a type error with the return value